utm.yaml

```diff
@@ -1317,6 +1271,213 @@ components:
+          allOf:
+          - $ref: '#/components/schemas/Time'
+        response_body:
+          type: string
+          description: Base64-encoded body content sent/received in response to request.
+        response_code:
+          type: integer
+          format: int32
+          description: HTTP response code sent/received in response to request.
+
+    ErrorReport:
+      type: object
+      description: A report informing a server of a communication problem.
+      required:
+      - exchange
+      - problem
+      properties:
+        report_id:
+          type: string
+          maxLength: 128
+          description: ID assigned by the server receiving the report.  Not populated
+            when submitting a report.
+        exchange:
+          allOf:
+          - $ref: '#/components/schemas/RequestResponse'
+          description: The request (by this USS) and response associated with the error.
+        problem:
+          type: string
+          description: Human-readable description of the problem with the response.
+
+    ExchangeRecord:
+      type: object
+      description: |-
+        A record of data exchange (USS - USS or USS - DSS).
+      required:
+      - request_source
+      - exchange
+      properties:
+        request_source:
+          type: string
+          description: |-
+            A coded value that indicates the requesting USS: 'LOCAL' (this USS) or 'REMOTE' (peer USS)
+          enum:
+          - LOCAL
+          - REMOTE
+        exchange:
+          allOf:
+          - $ref: '#/components/schemas/RequestResponse'
+          description: |-
+            Time that the response was received or sent
+
+    PositionRecord:
+      type: object
+      description: |-
+        A record of vehicle telemetry information received by this USS (typically for conformance monitoring).
+      required:
+      - time
+      - telemetry
+      properties:
+        time:
+          allOf:
+          - $ref: '#/components/schemas/Time'
+          description: |-
+            Time that this position data was received by the (local) USS
+        telemetry:
+          $ref: '#/components/schemas/VehicleTelemetry'
+
+    OperationRecord:
+      type: object
+      description: |-
+        A record of a position data gathered through the course of a complete operation
+      required:
+      - positions
+      - operation_id
+      properties:
+        positions:
+          type: array
+          items:
+            $ref: '#/components/schemas/PositionRecord'
+          default: []
+        operation_id:
+          description: ID of the Operation associated with 'positions'
+          allOf:
+          - $ref: '#/components/schemas/EntityID'
+
+    PlanningRecord:
+      type: object
+      description: |-
+        A record of a single attempt to (successfully or unsuccessfully) create an Operational Intent.
+      required:
+      - time
+      - ovns
+      - operation_id
```

Comment:
`operation_id` should be optional as the USS may not have reached the step where it internally assigns an operation ID (that step might happen after a route has been determined, and the problem might be that the USS was unable to find a route)